### PR TITLE
Enable builds for POWER and ARM systems

### PR DESCRIPTION
**Pull request by carterbox**
I want to publish build of TomoPy for POWER systems (eventually). Scikit-build is one of the last dependencies that is not yet available, so this PR enables the build for POWER systems.

Related to: https://github.com/conda-forge/tomopy-feedstock/pull/19

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by carterbox** [comment]
@conda-forge-admin, please rerender

**Comment by jcfr** [comment]
@conda-forge-admin please rerender

**Comment by carterbox** [comment]
The test are failing with errors such as below:
```
assert 'Invalid CMakeLists.txt' in 'CMake Error: CMake was unable to find a
build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set. 
You probably need to select a different build tool.\n'

tests/test_broken_project.py:44: AssertionError
```
It seems that scikit-build is having trouble picking up Ninja as the first choice generator.

**Comment by carterbox** [comment]
All of these tests were failing because CMake could not get Unix Makefiles to work. 
```
tests/test_skbuild.py::test_generator[Unix Makefiles-make] FAILED        [ 84%]
tests/test_skbuild_variable.py::test_skbuild_variable_builds FAILED      [ 89%]
tests/test_skbuild_variable.py::test_skbuild_variable_wheel FAILED       [ 89%]
```
This is a problem because according to the [scikit-build docs](https://scikit-build.readthedocs.io/en/latest/generators.html), Ninja should be selected first. I know that Ninja is working correctly in the test environment because I forced one test to work by setting the `CMAKE_GENERATOR` environment variable.

**Comment by isuruf** [comment]
Can you please add packages to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/arch_rebuild.txt so that it builds the dependencies correctly?

File: recipe/run_test.sh [hunk 1]
```diff
@@ -1 +1,2 @@
+export CMAKE_GENERATOR=Ninja
```
Comment (isuruf) [inline recipe/run_test.sh]: Can you please remove this?

Comment (carterbox) [reply recipe/run_test.sh]: Removed.

**Comment by carterbox** [comment]
@jcfr, Thanks for starting the rerender for me! It appears that the scikit-build tests require that `make` is installed and maybe that `scikit-build` isn't correctly preferring Ninja over Unix Make. Let me know if you want an Issue opened over at scikit-build about this.

@isuruf,
> Can you please add packages to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/arch_rebuild.txt so that it builds the dependencies correctly?

You want me to add `tomopy` and `scikit-build` to this text file? (conda-forge/conda-forge-pinning-feedstock#652) Sorry, your request is a little unclear to me because I wasn't able to find any info about how migrations work.

**Comment by isuruf** [comment]
Can you please add aarch64 too?

**Comment by carterbox** [comment]
@isuruf, please rerender :laughing:

**Comment by isuruf** [comment]
? I'm not a bot

**Comment by carterbox** [comment]
Please ask the bot? It doesn't listen to me. :smile:

**Comment by isuruf** [comment]
Can you try again?

**Comment by carterbox** [comment]
@conda-forge-admin, please rerender

**Review by isuruf — APPROVED** [review]

**Comment by jcfr** [comment]
Thanks everyone for helping out :pray:

**Pull request merged**